data/pages-content/sponsor/information.md

```diff
@@ -56,126 +53,130 @@ For details, please refer to the relevant sections below.
 
+
 <details>
-  <summary><h2>Exhibition Schedule</h2></summary>
+  <summary><h3>Exhibition Schedule</h3></summary>
 
 **Exhibit Days**: during the three main conference days: **Wednesday to Friday,
-13-15 July 2022**.
+19-21 July 2023**
 
-**Exhibit Hours**: 9:00 - 18:00 on Wednesday & Thursday; 9:00 - 17:00 on Friday.
+**Exhibit Hours**: 9:00-17:30
 
 All booths should be staffed at least during the official breaks. It is highly
 recommended that they will be staffed throughout the opening hours, especially
-the more prominent ones, such as Keystone and Diamond.
-
+the more prominent ones, such as Keystone, Diamond and Platinum.
 </details>
 
 <details>
-  <summary><h2>Exhibit Booths</h2></summary>
+  <summary><h3>Exhibit Booths</h3></summary>
+
   Sponsors of Silver and above will all be assigned a booth, ranging from 6-56 sqm depending on the package.
 
-** Please refer to the
-[EuroPython 2022 - Exhibit Booths.pdf](https://drive.google.com/file/d/1k2m9t445G4tpMON1VlGIg-SSgYTsQy5m/view?usp=sharing)
+**Please refer to the
+[EuroPython 2023 - Exhibit Booths](https://drive.google.com/file/d/17A45zHpjwrvZxKkf8-e9yUsHSqV1p74E/view?usp=sharing)
 for mockups and what is included in your booth, as part of your sponsorship
-package. **
-
+package.**
 </details>
 
-<details>
-  <summary><h2>Booth Build and Teardown</h2></summary>
+<!-- <details>
+  <summary><h3>Booth Build and Teardown</h3></summary>
 
-- Build Time: Tuesday 8:00-16:00; sponsors can enter between 16:00-18:00
-- Teardown Time: Friday 17:00; nobody allowed after the teardown starts.
+- Build Time:   FIXME: Tuesday 8:00-16:00; sponsors can enter between 16:00-18:00
+- Teardown Time: Friday 17:30; nobody allowed after the teardown starts.
 
-</details>
+</details> -->
 
 <details>
-  <summary><h2>Booth Builder Contact</h2></summary>
-  We have contracted [OBExpo](http://www.obexpo.ie/) to set up and manage the booths. You can reach out to Tony O'Brien with your booth related questions: [tony@obexpo.ie](mailto:tony@obexpo.ie)
+  <summary><h3>Booth Builder Contact</h3></summary>
+
+  We have contracted [Exposale](https://www.exposale.eu/) to set up and manage the booths. You can reach out to [Linda Škarková](mailto:linda.skarkova@exposale.cz) with your booth related questions
 
 </details>
 
 <details>
-  <summary><h2 id="booth-images">Booth & Counter Graphic</h2></summary>
+  <summary><h3 id="booth-images">Booth & Counter Graphic</h3></summary>
 
-**Booth graphic**: All booths include a Wall Banner Graphic that covers the
+**Booth back wall graphic**: All booths include a wall graphic that covers the
 entire back wall of your booth. **It is highly recommended that you submit a
 company specific custom graphic for your booth. The pricing is already included
 in your package.**
 
-**You can upgrade to a single Stretched Graphic design. See
-[Booth Customisation](#booth-customisations) for details.**
+<!-- _Note: FIXME: these wall banners with your graphic can be taken down afterwards for
```

Comment:
`FIXME` can be removed, right?

Reply:
Yes! Removed! The comments were enough :)